Formula/w/web-ext.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class WebExt < Formula
+  desc "A command line tool to help build, run, and test web extensions"
+  homepage "https://github.com/mozilla/web-ext"
+  url "https://registry.npmjs.org/web-ext/-/web-ext-7.7.0.tgz"
+  sha256 "482e9eab19c3df4cd353c20bc099c483bf0587a6120482ea811216cf7a02136a"
+  license "MPL-2.0"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    (testpath/"manifest.json").write <<~EOF
+      {
+        "manifest_version": 2,
+        "name": "minimal web extension",
+        "version": "0.0.1"
+      }
+    EOF
+    system "#{bin}/web-ext lint"
```

Comment:
can we assert the response?

```suggestion
    system bin/"#{bin}/web-ext", "lint"
```

Reply:
The output is this.
```
Validation Summary:

errors          0              
notices         0              
warnings        0
```
I am checking how to assert that.

Reply:
Done.